docs/UserGuide.md

```diff
@@ -169,7 +168,7 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [role/ROLE] [t/TAG]…​`
+Format: `edit i/ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​` (other fields specific to the role)
```

Comment:
Should we write them out like [pref/PREFERENCES] [prod/PRODUCT] [skills/SKILL]? and this sounds like we dont allow them to change role, is that the case?